### ISSUE TITLE
Literature study on different keyword extraction algorithms

### ISSUE THREAD

**Issue by BALaka-18** [BALaka-18/rake_new2#27]
## Description
Create an extensive and in-depth literature study on various keyword extraction algorithms other than RAKE and Tf-Idf. Every algorithm must be accompanied by brief logical / mathematical explanation + examples (in text or in the form of pictures / diagrams)

## File structure
Create a ```Literature_Survey.md``` file in the root directory.

## Acceptance Criteria
- [ ] All instructions provided in the Description must be strictly followed.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Time Estimation
5 days - 1 week.

**Comment by theodumont** [comment]
Hey, I'd like to work on this! Is it ok?

**Comment by BALaka-18** [comment]
@theodumont Sure. Assigning it to you.

**Comment by BALaka-18** [comment]
@theodumont are you working on it ?

**Comment by theodumont** [comment]
Yes I am. I am documenting myself a bit and I'll try to have a first version soon!

**Comment by theodumont** [comment]
Do you have any preference on how I should render mathematical equations? I was going to use [codecogs](https://www.codecogs.com/latex/eqneditor.php) to insert html code. Is it ok for you?

**Comment by BALaka-18** [comment]
> Do you have any preference on how I should render mathematical equations? I was going to use [codecogs](https://www.codecogs.com/latex/eqneditor.php) to insert html code. Is it ok for you?

Yes it's absolutely fine with me.

**Comment by theodumont** [comment]
Is it ok for you if I do a PR now and mark it as a WIP? I'll commit frequently so that you can follow the advancement.

**Comment by BALaka-18** [comment]
> Is it ok for you if I do a PR now and mark it as a WIP? I'll commit frequently so that you can follow the advancement.

Yup, go ahead.

**Comment by theodumont** [comment]
I have some notes ready to be added to the `.md` file, I'll do this soon!

**Comment by theodumont** [comment]
I've added a portion of my notes. Don't hesitate to tell me if the style and tone suits you!

**Comment by anirudhsai20** [comment]
Hi @BALaka-18 . Can I please work on this under DWOC?
